src/ignitus-Profile/ignitus-StudentProfile/styles.ts

```diff
@@ -0,0 +1,221 @@
+import styled from '@emotion/styled';
+import {Container, flexibleColDiv, flexibleRowDiv} from '../../ignitus-Shared/ignitus-DesignSystem/shared';
+import Icon from '../../ignitus-Shared/ignitus-Utilities/Components/icon';
+import * as C from '../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors';
+import { minimumWidthQuery } from '../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/media';
+import * as F from '../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/fonts';
+import {Avatar} from '../../ignitus-AboutPage/ignitus-Team/Styles';
+
+
+
+export const Container1 = styled(Container)`
+  margin: 10rem auto;
+  max-width: ${minimumWidthQuery[3]};
+  height: auto;
+  background: ${C.White};
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
+  border-radius: 30px;
+  padding-left: 5rem;
+  padding-right: 0;
+  padding-bottom: 5rem;
+`;
+
+export const Container2 = styled.div`
+  margin-right: -0.3rem;
+  margin-top: -0.3rem;
+  float: right;
+`;
+
+export const Container3 = styled.div`
+  padding-top: 3rem;
+  display: flex;
+  flex-direction:column;
+  justify-items: space-around;
+
+`;
+
+export const ElementContainer = styled(Container)`
+  max-width: ${minimumWidthQuery[3]};
+  flex:1;
+
+`;
+
+export const HeadingContainer = styled(flexibleRowDiv)`
+  // padding-top: 3rem;
+`;
+
+export const NameContainer = styled.div`
+  padding: 1rem;
+  flex: 10;
+`;
+
+export const StyledIcon = styled(Icon)`
+  height: 2rem;
+  width: 2rem;
+  fill: ${C.IgnitusBlue};
+  transition: 0.3s;
+  flex: 1;
+`;
+
+export const ContentContainer = styled(Container)`
+  width: 90%;
+  float: right;
+`;
+
+export const AboutContent = styled.textarea`
+  font-size: ${F.SM};
+  outline: none;
+  border:none;
+  width: 100%;
+  overflow: hidden;
+
+`;
+
+export const Education1 = styled.input`
+  font-size: ${F.SM};
+  font-weight: ${F.SemiBold};
+  outline:none;
+  border:none;
+  width: 15rem;
+`;
+
+export const Education2 = styled.input`
+  font-size: ${F.SM};
+  outline:none;
+  border:none;
+  width: 15rem;
+`;
+
+export const Education3 = styled.input`
+    font-size: ${F.SM};
+    outline:none;
+    border:none;
+    width: 15rem;
+`;
+
+export const Publication = styled(flexibleRowDiv)`
+  align-items: flex-start;
+`;
+
+export const PubInput = styled.textarea`
+  font-size: ${F.XS};
+  outline:none;
+  border:none;
+  width: 100%;
+  flex:20;
+  overflow:hidden;
+`;
+
+export const PubStyledIcon = styled(Icon)`
+  height: 1.3rem;
+  width: 1.3rem;
+  fill: ${C.IgnitusBlue};
+  transition: 0.3s;
+  flex: 1;
+`;
+
+export const ContriContent = styled.div`
+  display: flex;
+  flex-direction:row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+`;
+
+export const ContriElement = styled(flexibleRowDiv)`
```

Comment:
Please try to use understandable convention names instead of `Contri`

Reply:
sure, I'll edit that